specs/protocol/tx_validity.md

```diff
@@ -106,7 +106,7 @@ def unavailable_balance(tx, col) -> int:
     """
     sentBalance = sum_outputs(tx, col)
     gasBalance = gasPrice * gasLimit
-    bytesBalance = size(tx) * GAS_PER_BYTE * gasPrice
+    bytesBalance = size(tx) * bytePrice
```

Comment:
Can you

1. remove the `GAS_PER_BYTE` entry in the constants table in `consensus.md`
2. clarify in a comment that `size()` does not include witness data (otherwise the block proposer could just include a bunch of witness data and make users pay for it)
3. fix the correct change validity rules to account for cost of bytes

?

Reply:
Addressed 1 & 2. For 3 I didn't explicitly mention the bytes since their cost is already implicitly included in the initial free balance.